EventFilter/HcalRawToDigi/src/HcalUnpacker.cc

```diff
@@ -582,17 +582,62 @@ void HcalUnpacker::unpackUTCA(const FEDRawData& raw, const HcalElectronicsMap& e
     int nps=(amc13->AMCId(iamc)>>12)&0xF;
     
     HcalUHTRData uhtr(amc13->AMCPayload(iamc),amc13->AMCSize(iamc));
+    //debug printouts
+    //int nwords=uhtr.getRawLengthBytes()/2;
+    //for (int iw=0; iw<nwords; iw++) 
+    //    printf("%04d %04x\n",iw,uhtr.getRawData16()[iw]);
```

Comment:
generally, all commented out code should not be present.
If it's useful, it can be hidden by ifdefs, or converted to LogTrace or LogDebug.